src/bundle/Resources/public/scss/_main-nav.scss

```diff
@@ -13,28 +13,28 @@
     }
 
     .nav-tabs {
-        background-color: $ez-ground-base-light;
+        background-color: $ez-ground-base-medium;
         border: none;
         margin-bottom: 0;
 
         .nav-link {
             padding: 1rem 2rem;
-            color: $ez-color-base-dark;
+            color: $ez-black;
             display: flex;
```

Comment:
It makes me think that we're missing the `transition: all .3s <timing-function>;` property here, to make the color changes nicer for end user.